src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv

```diff
@@ -0,0 +1,80 @@
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/FileUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileUtils.java","1138","50","1138","64"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/FileUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileUtils.java","1139","51","1139","65"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/filefilter/MagicNumberFileFilter.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/filefilter/MagicNumberFileFilter.java","264","79","264","114"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/output/TeeOutputStream.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/output/TeeOutputStream.java","32","38","32","70"
+"Commented-out code","Commented-out code makes the remaining code more difficult to read.","warning","This comment appears to contain commented-out code.","src/main/java/org/apache/commons/io/input/BOMInputStream.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/input/BOMInputStream.java","290","9","293","54"
+"Confusing overloading of methods","Overloaded methods that have the same number of parameters, where each pair of corresponding parameter types is convertible by casting or autoboxing, may be confusing.","recommendation","Method IOUtils.toInputStream(..) could be confused with overloaded method [[""toInputStream""|""relative://src/main/java/org/apache/commons/io/IOUtils.java:1521:59539:13""]], since dispatch depends on static types.","src/main/java/org/apache/commons/io/IOUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/IOUtils.java","1472","31","1472","43"
```

Comment:
>relative: [](start = 319, length = 9)

Interesting convention to resolve a problem we are well aware of. :) Looking forward to seeing how you handle this in the converter.